app/src/main/res/values-es/strings.xml

```diff
@@ -415,27 +415,26 @@
     <string name="changeIconDialogMessage">La aplicación puede cerrarse para actualizar los cambios. Vuelve después de que hayas admirado tu nuevo y atractivo icono.</string>
 
     <!-- New dashboard / menu buttons -->
-    <string name="manageWhitelist">Administrar lista blanca</string>
+    <string name="manageWhitelist">Gestionar Protección</string>
     <string name="reportBrokenSite">Reportar sitio web roto</string>
 
     <!-- Dialogs -->
     <string name="dialogAddTitle">Añadir</string>
     <string name="dialogEditTitle">Editar</string>
 
     <!-- Whitelist -->
-    <string name="whitelisetActivityTitle">Lista blanca</string>
-    <string name="whitelistAdd">Agregar a la lista blanca</string>
-    <string name="whitelistRemove">Eliminar de la lista blanca</string>
-    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio web de la lista blanca %s</string>
-    <string name="whitelistEntryDeleteConfirmMessage">¿Estás seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt; de la lista blanca?</string>
-    <string name="whitelistExplanation">Estos sitios web incluidos en la lista blanca no se actualizarán por Protección de Privacidad</string>
-    <string name="whitelistNoEntries">Aún no hay sitios web en la lista blanca</string>
+    <string name="whitelisetActivityTitle">Gestionar Protección</string>
+    <string name="whitelistAdd">Habilitar Protección</string>
+    <string name="whitelistRemove">Deshabilitar Protección</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio desprotegido %s</string>
+    <string name="whitelistEntryDeleteConfirmMessage">Seguro que quieres habilitar de nuevo Protección de Privacidad para el sitio &lt;b>%s&lt;/b>?</string>
+    <string name="whitelistExplanation">La Protección de Privacidad no está aplicada a estos sitios</string>
+    <string name="whitelistNoEntries">Protección de Privacidad está habilitado para todos los sitios.</string>
     <string name="whitelistDomainHint">www.ejemplo.com</string>
-    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>ejemplo.com</b>
-  </string>
+    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>ejemplo.com</b></string>
 
     <!-- Settings -->
-    <string name="settingsPrivacyProtectionWhitelist">Lista blanca de Protección de Privacidad</string>
+    <string name="settingsPrivacyProtectionWhitelist">Sitios desprotegidos</string>
```

Comment:
The same than in a previous comment. it sounds better to me `Sitios no protegidos` than `desprotegidos`.